src/main/java/com/compatibility/Compatibility.java

```diff
@@ -0,0 +1,35 @@
+package com.compatibility;
+
+import com.compatibility.tinkers.ToolBrokenCheck;
+import net.minecraft.item.ItemStack;
+import org.jetbrains.annotations.Nullable;
+
+/**
+  * This class is to store the methods that call the methods
+  * to check for miscellaneous compatibility problems
+  */
+public class Compatibility
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this class "final" or add a public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS2974)